src/Game/Games/ConnectFour/ConnectFourView/ConnectFourView.java

```diff
@@ -54,6 +54,12 @@ public void setPawnView(Color color, Coord coord) {
         this.repaint();
     }
 
+    public void updateCurrentPlayer(Player currentPlayer){
+        if(currentPlayer != this.currentPlayer.getPlayer()){
```

Comment:
changer le nom de this.currentPlayer car on comprend pas que c'est un `DisplayPanel` en fait